### ISSUE TITLE
#### parsed to <h1>

### ISSUE THREAD

**Issue by vedam** [developit/snarkdown#47]
hi,
trying to understand the regex, I failed glorious.   ;-)

I noticed:
```
#### is parsed to <h1>
##### is parsed to <h2>
###### is parsed to <h3>
```

easily tested in https://snarky.surge.sh/
Is that intended?

**Comment by developit** [comment]
Ah - looks like the repeat is wrong.

**Comment by keul** [comment]
I can confirm a bug, or something that seems related to this.

I can't make snarkdown work with `####` or more, it put an empty H3, then another head with the text inside.

**Comment by vedam** [comment]
@Jonarod Yes I think so. I didn't see that, sorry.

**Comment by def14nt** [comment]
@Jonarod still busted mate

**Comment by Jonarod** [comment]
@def14nt this PR has not been merged as far as I understand.
I copy-pasted my PR in a fiddle here: [https://jsfiddle.net/ohbg9zpk/](https://jsfiddle.net/ohbg9zpk/) it works for me :)

**Comment by mesqueeb** [comment]
@Jonarod looks merged to me!
Is this issue to be resolved & closed?

**Comment by talentlessguy** [comment]
Still happens on `snarkdown` 1.2.2

**Comment by developit** [comment]
This fixed has been released as [2.0.0](https://github.com/developit/snarkdown/releases/tag/2.0.0).

(here's @Jonarod's [fiddle](https://jsfiddle.net/developit/tzsjfho0/) with that version)
